tests/TestCase/ORM/Association/BelongsToTest.php

```diff
@@ -461,7 +461,7 @@ public function testAttachToFormatResultsNoDirtyResults(): void
     {
         $this->setAppNamespace('TestApp');
         $articles = $this->getTableLocator()->get('Articles');
-        $articles->belongsTo('Authors')
```

Comment:
Not sure users would ever have this code in their table init, but I think the error makes sense in this situation.